apps/nextjs/src/components/AppComponents/Chat/sidebar-item.tsx

```diff
@@ -1,53 +1,51 @@
 "use client";
 
 import * as React from "react";
+import { useState } from "react";
 
-import { motion } from "framer-motion";
+import { OakFlex, OakSpan } from "@oaknational/oak-components";
 import Link from "next/link";
-import { usePathname } from "next/navigation";
 
 import AiIcon from "@/components/AiIcon";
-import { buttonVariants } from "@/components/AppComponents/Chat/ui/button";
 import { IconUsers } from "@/components/AppComponents/Chat/ui/icons";
+import { SheetTrigger } from "@/components/AppComponents/Chat/ui/sheet";
 import {
   Tooltip,
   TooltipContent,
   TooltipTrigger,
 } from "@/components/AppComponents/Chat/ui/tooltip";
+import BinIcon from "@/components/BinIcon";
 import type { SideBarChatItem } from "@/lib/types";
-import { cn } from "@/lib/utils";
 
+import { useDialog } from "../DialogContext";
 import { constructChatPath } from "./Chat/utils";
 
 interface SidebarItemProps {
   chat: SideBarChatItem;
   children?: React.ReactNode;
 }
 
-export function SidebarItem({ chat, children }: SidebarItemProps) {
-  const pathname = usePathname();
-
-  const isActive = pathname.includes(chat.id);
-
+export function SidebarItem({ chat }: SidebarItemProps) {
+  const { setDialogWindow } = useDialog();
+  const [isHovered, setIsHovered] = useState(false);
+  function deleteChatById() {
+    localStorage.setItem("chatIdToDelete", chat.id);
+    setDialogWindow("clear-single-chat");
+  }
   return (
-    <motion.div
-      className="relative h-18"
-      variants={{
-        initial: {
-          height: 0,
-          opacity: 0,
-        },
-        animate: {
-          height: "auto",
-          opacity: 1,
-        },
-      }}
-      transition={{
-        duration: 0.25,
-        ease: "easeIn",
-      }}
+    <OakFlex
+      $justifyContent="space-between"
+      $alignItems="center"
+      $background={isHovered ? `grey30` : `white`}
```

Comment:
Surely Oak components provide a way to do this with CSS? Coudl we use styled-components if not?

Reply:
This is a bit of a hack sure its to match the current functionality which had a slightly weird hover